<ui version="4.0" >
 <class>Widget</class>
 <widget class="QWidget" name="Widget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="18" column="1" >
       <widget class="QComboBox" name="energyLayoutCmb" />
      </item>
      <item row="18" column="0" >
       <widget class="QLabel" name="label_19" >
        <property name="text" >
         <string>Energy layout:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>energyLayoutCmb</cstring>
        </property>
       </widget>
      </item>
      <item row="17" column="0" >
       <widget class="QLabel" name="label_17" >
        <property name="text" >
         <string>Mutation rate:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>mutationEd</cstring>
        </property>
       </widget>
      </item>
      <item row="16" column="1" >
       <widget class="QLineEdit" name="stackSizeEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="16" column="0" >
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>Stack size:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>stackSizeEd</cstring>
        </property>
       </widget>
      </item>
      <item row="15" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="text" >
         <string>Processor steps:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>stepsEd</cstring>
        </property>
       </widget>
      </item>
      <item row="15" column="1" >
       <widget class="QLineEdit" name="stepsEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="17" column="1" >
       <widget class="QLineEdit" name="mutationEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="14" column="1" >
       <widget class="QLineEdit" name="heightEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="13" column="1" >
       <widget class="QLineEdit" name="widthEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="12" column="1" >
       <widget class="QLineEdit" name="minTotalNodeEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="11" column="1" >
       <widget class="QLineEdit" name="minTotalBugEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="10" column="1" >
       <widget class="QLineEdit" name="totalEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1" >
       <widget class="QLineEdit" name="maxNodeEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1" >
       <widget class="QLineEdit" name="minNodeEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1" >
       <widget class="QLineEdit" name="stayEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="moveEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="childEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="maxBugEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="minBugEnergyEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="maxAgeEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="minDataEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="maxDataEd" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Max data size:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>maxDataEd</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Min data size:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>minDataEd</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Max age:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>maxAgeEd</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Min bug energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>minBugEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Max bug energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>maxBugEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Child energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>childEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Move energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>moveEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Stay energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>stayEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Min node energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>minNodeEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Max node energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>maxNodeEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Total energy:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>totalEnergyEd</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Min bug investment:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>minTotalBugEd</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Min node investment:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>minTotalNodeEd</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Width:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>widthEd</cstring>
        </property>
       </widget>
      </item>
      <item row="14" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Height:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>heightEd</cstring>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2" >
       <widget class="QCheckBox" name="chkKillOnEdge" >
        <property name="layoutDirection" >
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text" >
         <string>Kill on Edge</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="" >
   <property name="text" >
    <string/>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>minDataEd</tabstop>
  <tabstop>maxDataEd</tabstop>
  <tabstop>maxAgeEd</tabstop>
  <tabstop>minBugEnergyEd</tabstop>
  <tabstop>maxBugEnergyEd</tabstop>
  <tabstop>childEnergyEd</tabstop>
  <tabstop>moveEnergyEd</tabstop>
  <tabstop>stayEnergyEd</tabstop>
  <tabstop>minNodeEnergyEd</tabstop>
  <tabstop>maxNodeEnergyEd</tabstop>
  <tabstop>totalEnergyEd</tabstop>
  <tabstop>minTotalBugEd</tabstop>
  <tabstop>minTotalNodeEd</tabstop>
  <tabstop>widthEd</tabstop>
  <tabstop>heightEd</tabstop>
  <tabstop>stepsEd</tabstop>
  <tabstop>stackSizeEd</tabstop>
  <tabstop>mutationEd</tabstop>
  <tabstop>energyLayoutCmb</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
